src/Apps/Conversation/Components/Message.tsx

```diff
@@ -30,29 +39,54 @@ const Attachment: React.FC<AttachmentProps> = props => {
     return null
   }
 }
-interface MessageProps {
+interface MessageProps extends Omit<BoxProps, "color"> {
   message: Message_message
   initialMessage?: string
   isFirst: boolean
+  showTimeSince?: boolean
 }
 const Message: React.FC<MessageProps> = props => {
-  const { message, initialMessage, isFirst } = props
-  const createdAt = DateTime.fromISO(message.createdAt).toRelative()
+  const { message, initialMessage, isFirst, showTimeSince, ...boxProps } = props
+  const { isFromUser, body } = message
+  const text = isFirst ? initialMessage : body
+  // const createdAt = DateTime.fromISO(message.createdAt).toRelative()
```

Comment:
Did you mean to leave these comments in?

Reply:
Yep! I'm going to be follow up w/ another PR to clean up + add tests.